skimage/measure/__init__.py

```diff
@@ -7,7 +7,7 @@
 from ._pnpoly import points_in_poly, grid_points_in_poly
 from ._moments import moments, moments_central, moments_normalized, moments_hu
 from .profile import profile_line
-from .fit import LineModel, CircleModel, EllipseModel, ransac
+from .fit import LineModel, LineModel3D, CircleModel, EllipseModel, ransac
```

Comment:
@kevin-keraudren Do you think it is possible to dispatch to the correct dimension from the input data?